# Marker selected to load
uint32[] id
---
bool ret
